apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -124,11 +123,7 @@ const useScrub = ({
     const scrub = numericScrubControl(scrubRef.current, {
       initialValue: valueRef.current.value,
       onValueInput(event) {
-        onChangeRef.current({
-          type,
-          unit,
-          value: event.value,
-        });
+        if (inputRef.current) inputRef.current.value = String(event.value);
```

Comment:
We already have inputRef.current null check, if `ts` can't detect that probably would be better to define at top
`const inputElement  = inputRef.current`

Also, I think in case of any ref is null inside effect you need to at least `warnOnce` or even `throw new Error ` instead of just `return ""`